examples/pnp_demo/src/main.rs

```diff
@@ -143,11 +158,21 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     )?;
 
     println!("Ground truth translation: {:?}", gt_t);
-    println!("Estimated translation  : {:?}", result.translation);
+    println!("Estimated translation (LM): {:?}", result_lm.translation);
     println!("Ground truth rotation   : {:?}", gt_r);
-    println!("Estimated rotation:\n{:?}", result.rotation);
-    if let Some(rmse) = result.reproj_rmse {
-        println!("Reprojection RMSE: {:.3} px", rmse);
+    println!("Estimated rotation (EPnP):\n{:?}", result_epnp.rotation);
+    println!("Estimated rotation (LM):\n{:?}", result_lm.rotation);
+    if let Some(rmse) = result_epnp.reproj_rmse {
```

Comment:
@cjpurackal i feel the result of pnp should be Optional. And then inside if lm was requested, this being also optional. Every attribute being optional is not a good pattern